UPGRADE-6.3.md

```diff
@@ -435,12 +435,17 @@ is recommended (although this tool may not find database-related deprecations).
 - chameleon_system_debug: backtrace_enabled
 - chameleon_system_debug: backtrace_limit
 
+## Bundles
+
+- RevisionManagementBundle
```

Comment:
Could you also add a section where the deprecation is explained (above the deprecation section)? Should developers remove the bundle from AppKernel?

Reply:
done

Reply:
Let me elaborate: The section where deprecations are noted should be strictly limited to be a reference without further information. Users can see here which entities are deprecated and can quickly view if this is a problem for their current code. If there's anything requiring immediate action, this information should be in a separate section (while information on replacements etc is written in code; there it can be gathered by tools like the recommended deprecation detector). And as the deprecation list is intended to be the last section in the document, please add this section above this one.